core/components/atoms/select/select.js

```diff
@@ -12,6 +13,22 @@ const selectOpacity = {
 }
 const PLACEHOLDER_VALUE = '__select_placeholder'
 
+const StyledIcon = styled(Icon).attrs({
+  name: 'dropdown-fill',
+  size: '14'
+})`
+  position: absolute;
+  right: 12px;
+
+  svg {
+    display: block;
+
+    path {
+      fill: ${colors.text.default};
+    }
```

Comment:
Looks like this could be a `color` prop as well?

Reply:
Yes!